mongo/write.go

```diff
@@ -149,3 +150,20 @@ func (client Client) ReplaceOne(ctx context.Context, colname string,
 	}
 	return result.ModifiedCount, e
 }
+
+// BulkWrite - function to write on multiple documents
+// ctx can be nil, models are an array of operations to execute on the collection, opts is optional (allow to be not provided without breaking function in Go)
+func (client Client) BulkWrite(ctx context.Context, collname string, models []mongo.WriteModel, opts ...*options.BulkWriteOptions) (mongo.BulkWriteResult, error) {
```

Comment:
on the other functions, the opts was not considered as optional (...)
should we standardize?

Reply:
yes, we should use ... to allow for multiple options to be provided (if necessary) (currently only [two](https://www.mongodb.com/docs/drivers/go/current/fundamentals/crud/write-operations/bulk/#modify-behavior))

if nil, it should then use the driver's default behaviour, depending on the operation to perform